content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md

```diff
@@ -0,0 +1,282 @@
+---
+title: "Install and Manage MAAS Gateway"
+metaTitle: "Install and Manage MAAS Private Cloud Gateway"
+metaDescription: "Learn how to install and manage the MAAS Private Cloud Gateway in Palette."
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+
+# Overview 
+
+The Private Cloud Gateway (PCG) enables support for private cloud or data center environments. You can set up the cloud gateway as a single-node or three-node cluster based on your requirements for high availability (HA).  
+<br />
+
+<InfoBox>
+
+For production environments, we recommend using three nodes. If you initially set up the gateway with one node, you can resize it at a later time. 
+
+</InfoBox>
+
+
+Palette provides an installer in the form of a Docker container that is temporarily deployed on your laptop, workstation or jump box. You can use the installer on any linux system that has a Docker daemon installed and connectivity to Palette and the MAAS identity endpoint. 
+
+<br />
+
+
+<WarningBox>
+
+The installer does not currently work on MacOS.
+
+</WarningBox>
+
+
+## Prerequisites
+
+- Canonical [MAAS installed](https://maas.io/docs/how-to-install-maas), set up, and available in your environment.
+
+
+- A linux computer with a Docker daemon installed and a connection to Palette and the MAAS endpoint. The installer must be invoked on a linux system. We have tested the gateway installation using Ubuntu 20.04.
+
+
+- One IP address for a Kubernetes control plane.
```

Comment:
Each node needs an IP and then we allocate an additional IP for the control plane's API server. I propose this wording:

Private cloud gateway IP requirements:
- One IP address per node: one for a single-node gateway or three for a three-node gateway.
- One IP for the Kubernetes control plane.

Reply:
I assume we mean the PCG cluster's control plane? 
How is that IP obtained - is it via DHCP? & is it a virtual IP, since in a 3 node cluster I assume all 3 are master nodes? sorry but I am still confused over this IP address and I think readers will be too.

Reply:
I would defer to @sadysnaat on that. I think we use [kube-vip in ARP mode](https://kube-vip.io/docs/modes/arp/) to handle the control plane VIP. But Deepak would know best.